MekHQ/src/mekhq/campaign/parts/PartInUse.java

```diff
@@ -35,13 +35,14 @@ public PartInUse(Part part) {
             appendDetails(sb, part);
         }
         part.setUnit(u);
-        this.id = part.getId();
+        this.name = part.getName();
```

Comment:
How does this play with loading files from the previous version?

Reply:
So the PartInUse is not actually stored in save files, it's just a (lossy) model constructed for this dialog that is deleted when the dialog is closed. There should be no impact on save files/backwards compatibility.

Reply:
Ok. If PartInUse isn't being saved, then we can consider this good to go.

Reply:
Alright yeah, I just double checked and it's not saved anywhere. The only part that is persistent is the `IAcquisitionWork partToBuy` field, which is pulled from the `Part` this is being constructed from and saved to the shopping list when buying items, but neither PR touched that object. So we should be good.